datarepo-clienttests/src/main/java/scripts/testscripts/DRSLookup.java

```diff
@@ -32,6 +32,10 @@
 import scripts.testscripts.baseclasses.SimpleDataset;
 import scripts.utils.DataRepoUtils;
 
+/*
+XXX: if making any changes to this class make sure to notify the #dsp-batch channel! Describe the change and any
```

Comment:
For block comments, each line must start with a `*`. Or use a `//` comment. https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Also is XXX what was decided on for this? I would expect to see `NOTE`. To me XXX means a value that needs to be replaced with something else later on.

Reply:
I hesitate to use `NOTE` since we use it in other places in our codebase, but I switched to using blockstyle comments.

Reply:
I opened a larger conversation about that internally.